docs/rst/notebook-traditional.ipynb

```diff
@@ -0,0 +1,23 @@
+{
+ "cells": [
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "# Notebook (traditional)\n",
+    "\n",
+    "This is just a placeholder.\n",
+    "See {ref}`notebook-traditional` instead."
+   ]
```

Comment:
This cell's Markdown links to it, using
```
{ref}`notebook-traditional`
```

In this spirit, it demonstrates that notebooks are real documents, thus become parts of Sphinx' document tree, so they can use all the features, including linking between them as if they were traditional reStructuredText or Markdown documents.